tests/docs-screenshots.spec.ts

```diff
@@ -91,20 +117,592 @@ test("instances", async ({ page }) => {
+  await page.waitForTimeout(1000);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/yaml_configuration.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+  await page.getByRole("button", { name: "Cancel" }).click();
+
+  await page.getByTestId("tab-link-Terminal").click();
+  await page.waitForTimeout(2000);
+  await page.keyboard.type("rm /usr/bin/bash");
+  await page.keyboard.press("Enter");
+  await page.reload();
+  await page.waitForSelector(`text=The connection was closed abnormally`);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/broken_terminal.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  // -----------------------------------------------------
+  // Desktop vm
+
+  const vminstance = "Ubuntu-desktop";
+  await gotoURL(page, "/ui/");
+  await page.getByText("Instances", { exact: true }).click();
+  await page.getByText("Create instance").click();
+  await page.getByPlaceholder("Enter name").fill(vminstance);
+  await page.getByRole("button", { name: "Browse images" }).click();
+  await page
+    .locator("tr")
+    .filter({
+      hasText: "Ubuntunobledesktopvirtual-machineubuntu/noble/desktopLXD",
+    })
+    .getByRole("button")
+    .click();
+
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/create_desktop_vm.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+  await page.getByRole("button", { name: "Create", exact: true }).click();
+
+  await visitInstance(page, vminstance);
+  await page.getByTestId("tab-link-Configuration").click();
+
+  await page.getByText("Disk").click();
+  await page.getByRole("button", { name: "Create override" }).click();
+  await page.getByPlaceholder("Enter value").fill("30");
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/root_disk_size.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  // desktop_console.png and hello_world_desktop.png may nee to be screenshotted automatically as it take too long for the instance to start up by itself. The test times out.
+
+  // // Turn on Secure Boot in order to start the VM.
+  // await page.getByText("Security policies").click();
+  // await setOption(page, "Enable secureboot", "false");
+  // await page.getByRole("button", { name: "Save 2 changes" }).click();
+  // await page.getByTestId("notification-close-button").click();
+  // await page.waitForTimeout(1000);
+
+  // // Desktop console
+  // await visitAndStartInstance(page, vminstance);
+  // await page.getByTestId("tab-link-Console").click();
+  // await page.screenshot({
+  //   path: "tests/screenshots/doc/images/tutorial/desktop_console.png",
+  //   clip: getClipPosition(0, 0, 1440, 760),
+  // });
+
+  // // Hello world desktop
+  // await page.screenshot({
+  //   path: "tests/screenshots/doc/images/tutorial/hello_world_desktop.png",
+  //   clip: getClipPosition(0, 0, 1440, 760),
+  // });
```

Comment:
Can this be removed? Or should it be uncommented?

Reply:
How about putting these two into a dedicated test suite? Then the timeout of 2 minutes might not be hit.

Also, I think we should have the click path as part of the test that opens the file browser and goes to the folder as well as creating necessary files in the screen

Reply:
Let's isolate this into a dedicated test, and then we can decide how to run it. Either manually running with the prerequisite of increasing the timeout before. Or we could in CI script the timeout extension for this very test.

Reply:
Did some testing and with a cached desktop image the boot only takes ~40 seconds. So should be within the timeout of two minutes if the image is cached. I think we can make it a prerequisite, the desktop image is cached. And as we schedule the job later in CI, we can add a pre-run command that is downloading the image.